automation/release.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -xe
+
+# This script tags current version and create a new release at github and pushes
+# to quay.io
+# To run it just do proper docker login and pass the git hub user password as
+# GIT_USER and GIT_PASSWORD env variables.
+
+# To pass user/password from automations, idea is taken from [1]
+# [1] https://stackoverflow.com/questions/8536732/can-i-hold-git-credentials-in-environment-variables
```

Comment:
nice
looking on
https://stackoverflow.com/questions/8536732/can-i-hold-git-credentials-in-environment-variables

isnt the last idea there is more stable ? (without sleep and race cond)

Reply:
Didn't want to add another script just to print the password